packages/realm/src/schema/types.ts

```diff
@@ -91,7 +92,7 @@ export type PropertiesTypes = {
  */
 export type ObjectSchemaProperty = {
   type: PropertyType;
-  objectType?: string;
+  objectType?: string; // TODO: Rename to elementType
```

Comment:
I believe there was a plan before to rename `ObjectSchemaProperty` to `PropertySchema`. If going ahead with the rename of its property `objectType` to `elementType`, wouldn't it be best to also go ahead with the rename to `PropertySchema` at the same time? So that users can deal with the renames fewer times.

(In that case, `CanonicalObjectSchemaProperty` would most likely also need to change to `CanonicalPropertySchema`).

This would require several changes across the code base.

Reply:
Oh right!

Reply:
Please an issue about the rename. Furthermore, add it to the "Breaking Changes v12" epic in Jira.

Reply:
Will do!